src/OrbitQt/CallTreeViewItemModel.cpp

```diff
@@ -91,6 +93,21 @@ QVariant CallTreeViewItemModel::GetDisplayRoleData(const QModelIndex& index) con
         return QString::fromStdString(
             absl::StrFormat("%.2f%%", unwind_errors_item->GetPercentOfParent()));
     }
+  } else if (unwind_error_type_item != nullptr) {
+    switch (index.column()) {
+      case kThreadOrFunction:
+        return QString::fromStdString(
+            orbit_client_data::CallstackTypeToString(unwind_error_type_item->error_type()));
+      case kInclusive:
+        return QString::fromStdString(absl::StrFormat(
+            "%.2f%% (%llu)",
+            unwind_error_type_item->GetInclusivePercent(call_tree_view_->sample_count()),
+            unwind_error_type_item->sample_count()));
+      // Exclusive makes no sense for this node, and would always be zero.
```

Comment:
I think that based on the comment on `AddUnwindErrorToTopDownThread` this will no longer be true.

Reply:
Now it is true again :)